robolectric/src/main/java/org/robolectric/util/ActivityController.java

```diff
@@ -178,6 +188,22 @@ public void run() {
       }
     });
 
+    ViewRootImpl root = component.getWindow().getDecorView().getViewRootImpl();
+    ShadowViewRootImpl shadow = (ShadowViewRootImpl) ShadowExtractor.extract(root);
+    Display display = Shadow.newInstanceOf(Display.class);
+    ShadowDisplay shadowDisplay = Shadows.shadowOf(display);
+    Rect frame = new Rect(0, 0, shadowDisplay.getWidth(), shadowDisplay.getHeight());
+    Rect insets = new Rect(0, 0, 0, 0);
+    if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
```

Comment:
@jongerrish please disregard the comments about viewRootImpl.java.vm I am in the process of using this pattern to avoid defining a bunch of functions in vm